meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -57,7 +57,7 @@ DEPENDS = " \
   "
```

Comment:
Small typo in the commit message "It represents the init process of
any container that docker needs to start.